### PR TITLE
WFP-1533, WFP-1524, WFP-1525: Improved tab titles

### PR DESCRIPTION
**Pull request by sp-watson**

### CODE REVIEW

File: app/routes/dashboard.js [hunk 1]
```diff
@@ -38,22 +39,22 @@ module.exports = function (router) {
...
-          linkId: req.params.id,
-          title: result.title,
-          subTitle: result.subTitle,
-          tabTitle: getTabTitle(result.title, subNav, organisationLevel),
-          breadcrumbs: result.breadcrumbs,
-          subNav,
-          date: result.date,
-          onOffenderManager: true
-
+      return getExport(id, organisationLevel).then(function (result) {
```
Comment (sp-watson) [inline app/routes/dashboard.js:42]: Fixing a bug: Export data was been retrieved async but handled as sync

File: app/routes/omic-export.js [hunk 2]
```diff
@@ -36,18 +38,20 @@ module.exports = function (router) {
...
-        organisationLevel,
-        linkId: req.params.id,
-        title: result.title,
-        subTitle: result.subTitle,
-        breadcrumbs: result.breadcrumbs,
-        subNav: getSubNav(id, organisationLevel, req.path, workloadTypes.OMIC, authorisedUserRole.authorisation, authorisedUserRole.userRole),
-        date: result.date,
-        onOmic: true
-
+      return getExport(id, organisationLevel).then(function (result) {
```
Comment (sp-watson) [inline app/routes/omic-export.js:41]: Fixing export bug as per dashboard.js

File: app/services/get-export.js [hunk 3]
```diff
@@ -9,11 +8,9 @@ module.exports = function (id, organisationLevel) {
     if (organisationalUnitType.name === 'hmpps') {
       title = organisationalUnitType.displayText
     }
-    const subTitle = navTitleConstants.OFFENDER_MANAGEMENT.displayText
```
Comment (sp-watson) [inline app/services/get-export.js:12]: Moved the subTitle up a level as it can be different for each caller

File: app/services/get-tab-title.js [hunk 4]
```diff
@@ -1,21 +1,24 @@
...
-module.exports = function (pageTitle, subNav, organisationLevel) {
-  let selectedSubNavTitle = ''
+module.exports = function (pageTitle, pageSubTitle, subNav, organisationLevel) {
+  let selectedSubNavTitle = null
   subNav?.forEach(function (item) {
     if (item.active) {
       selectedSubNavTitle = item.title
     }
   })
+  if (selectedSubNavTitle === 'OMIC Overview') {
```
Comment (sp-watson) [inline app/services/get-tab-title.js:10]: i don't like this but it's probably the easiest option. That is unless we can convince Rose it's inconsistent and change the H1 (or change what's shown in the tab title)

**Review by carlov20 — APPROVED** [review]

**Pull request merged**